scripts/launch_cluster.py

```diff
@@ -462,6 +478,7 @@ def _write_kube_config(self, cluster_name):
 
         # Write in YAML.
         config_text = yaml.dump(cluster_config, default_flow_style=False)
+        os.makedirs(os.path.dirname(self._output_path), exist_ok=True)
```

Comment:
I remerber that you also need to write a `aws-auth-cm.yaml`? That file also need to create its directory first.